init.c

```diff
@@ -408,6 +409,8 @@ int main(int argc, char *argv[])
 		exit_error(errno, "Could not init cgroup");
 	other_mounts();
 
+	load_apparmor_profiles();
```

Comment:
why here? just asking so I get who is using this etc. :)

load_apparmor_profiles would be executed in appengine mode, so it would be nice to have the aforementioned configurable path stuff

Reply:
That's OK. I was meaning more if we are calling load_apparmor_profiles as late as possible (Notice that we don't have logserver initialized and that's inconvinient for debugging, for example)